meson.build

```diff
@@ -467,7 +467,8 @@ uwp = cc.find_library('windowsapp', required: uwp_opt)
 features += {'uwp': uwp.found()}
 if features['uwp']
     dependencies += uwp
-    path_source = files('osdep/path-uwp.c')
+    path_source = files('osdep/path-uwp.c',
+                        'osdep/terminal-dummy.c')
```

Comment:
Maybe throw this down below with `subprocess-dummy.c` and `ipc-dummy.c` so it any theoretical thing that doesn't pass any of the other checks gets this file?

Reply:
Ok, done.

btw. I think this dummy thing should be removed along with uwp that no one is using.

Reply:
Fully agreed. But we can save that for later.